src/property-path.js

```diff
@@ -0,0 +1,91 @@
+
+const NodeSet = require('./node-set')
+
+function getPathObjects (graph, subject, path) {
```

Comment:
This function replaces https://github.com/zazuko/shacl-js/blob/master/src/rdfquery.js#L474